documentation/specs/src/economics/inflation-system.md

```diff
@@ -20,60 +20,60 @@ Inflation is calculated and paid per-epoch as follows.
 
 First, we start with the following fixed (governance-alterable) parameters:
 
-- $Cap_{PoS}$ is the cap of proof-of-stake reward rate, in units of percent per annum
+- $Cap_{PoS}$ is the cap of proof-of-stake reward rate, in units of percent per annum (genesis default: 10%)
 - $Cap_{SP-A}$ is the cap of shielded pool reward rate for each asset $A$, in units of percent per annum
-- $R_{PGF}$ is the public goods funding reward rate, in units of percent per annum
-- $R_{PoS-Target}$ is the target staking ratio (genesis default 2/3)
+- $I_{PGF}$ is the public goods funding reward rate, in units of percent per annum
+- $R_{PoS-Target}$ is the target staking ratio (genesis default: 2/3)
 - $R_{SP-A-Target}$ is the target amount of asset $A$ locked in the shielded pool (separate value for each asset $A$)
-- $EpochsPerYear$ is the number of epochs per year (genesis default 365)
-- ${KP}_{PoS}$ is the proportional gain of the proof-of-stake PD controller, as a fraction of the total input range
-- ${KD}_{PoS}$ is the derivative gain of the proof-of-stake PD controller, as a fraction of the total input range
-- ${KP}_{SP_A}$ is the proportional gain of the shielded pool reward controller for asset $A$, as a fraction of the total input range (separate value for each asset $A$)
-- ${KD}_{SP_A}$ is the derivative gain of the shielded pool reward controller for asset $A$, as a fraction of the total input range (separate value for each asset $A$) 
+- $EpochsPerYear$ is the number of epochs per year (genesis default: 365)
+- ${KP}_{PoS-nom}$ is the nominal proportional gain of the proof-of-stake PD controller, as a fraction of the total input range
+- ${KD}_{PoS-nom}$ is the nominal derivative gain of the proof-of-stake PD controller, as a fraction of the total input range
+- ${KP}_{SP_A-nom}$ is the nominal proportional gain of the shielded pool reward controller for asset $A$, as a fraction of the total input range (separate value for each asset $A$)
+- ${KD}_{SP_A-nom}$ is the nominal derivative gain of the shielded pool reward controller for asset $A$, as a fraction of the total input range (separate value for each asset $A$)
 
 Second, we take as input the following state values:
 
 - $S_{NAM}$ is the current supply of NAM
 - $L_{NAM}$ is the current amount of NAM locked in proof-of-stake
-- $I_{PoS}$ is the current proof-of-stake reward rate, in units of tokens per epoch
-- $E_{PoS-last}$ is the error in proof-of-stake lock ratio (stored from the past epoch)
+- $I_{PoS}$ is the current proof-of-stake reward rate, in units of percent per annum
```

Comment:
no, the proof-of-stake inflation controller value should be tokens per epoch, but the bounds should vary as percent per annum